testsuites/syncgateway/functional/topology_specific_tests/multiple_sync_gateways/test_scopes_and_collections.py

```diff
@@ -160,17 +182,20 @@ def test_scopes_and_collections_replication(scopes_collections_tests_fixture, pa
         continuous=False,
         collections_enabled=True
     )
-    admin_client_2.wait_until_sgw_replication_done(sg2["db"], replicator2_id, read_flag=True, max_times=3000)
+    admin_client_2.replication_status_poll(sg2["db"], replicator1_id, max_times=120)
 
     # 3. Check that the documents were replicated to sgw2
-    for i in range(0, num_of_docs - 1):
-        sg_client.get_doc(sg2_url, sg2["db"], uploaded_for_pull[i]["id"], rev=uploaded_for_pull[i]["rev"], auth=user2_auth, scope=scope, collection=collection)
+    sg2_collection_docs_ids = [row["id"] for row in sg_client.get_all_docs(url=sg2_admin_url, db=sg2["db"], auth=user2_auth, scope=scope, collection=collection)["rows"]]
+    assert_docs_replicated(uploaded_for_pull, sg2_collection_docs_ids, "sg2", sg2["db"], replicator1_id, "pull")
 
     # 4. Add another collection
     data1 = {"bucket": bucket, "scopes": {scope: {"collections": {collection: {"sync": sync_function}, collection2: {"sync": sync_function}}}}, "num_index_replicas": 0, "import_docs": True, "enable_shared_bucket_access": True}
     data2 = {"bucket": bucket2, "scopes": {scope: {"collections": {collection: {"sync": sync_function}, collection2: {"sync": sync_function}}}}, "num_index_replicas": 0, "import_docs": True, "enable_shared_bucket_access": True}
+    # collection_access = {scope: {collection: {"admin_channels": channels}, collection2: {"admin_channels": channels}}}
```

Comment:
Do we need this comment and the 2 below it?

Reply:
Removed